lang/ja.css

```diff
@@ -0,0 +1,5 @@
+:lang(ja) .theorem::before { content: '定理 ' counter(theorem) '. ' !important; }
+:lang(ja) .theorem {font-style: italic; }
+:lang(ja) .lemma::before { content: '補題 ' counter(theorem) '. ' !important; }
+:lang(ja) .proof::before { content: '証明: ' attr(title) !important; font-style: normal;}
```

Comment:
And a space before the last curly brace

Reply:
I do agree to your error fix.  
I'm not sure what is the right way on git to edit and fix them, as I am new to git. 
Would you please edit them, or if there is more appropriate way, would you please guide me.

Reply:
Absolutely! First off, congrats on you first pull request🥳! You should be able to edit the file on your local machine, commit it and then push it again to GitHub on the same branch. The changes will then appear here as well.

Reply:
Thanks.  
Now I am done.